### PR TITLE
Update auth provider

### PR DESCRIPTION
**Pull request by WeeMingQing**
# Implementation of auth kit

### CODE REVIEW

File: frontend/src/pages/Login.jsx [hunk 1]
```diff
@@ -22,15 +22,21 @@ import { useNavigate } from 'react-router-dom';
...
 import { useLogin } from '../hooks/mutations';
+import { isAuthenticated } from '../hooks/auth/authHook';
 
 export const Login = () => {
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (isAuthenticated() === 'true') {
+      navigate(URL_BASE_INV);
+    }
+  }, [navigate]);
```
Comment (jonasgwt) [inline frontend/src/pages/Login.jsx]: can we put this useEffect after all the const declerations

**Review by jonasgwt — COMMENTED** [review]
@WeeMingQing can we also remove react auth kit and relevant packages that we don't need from package.json

File: frontend/src/pages/Login.jsx (hunk 1 above)
Comment (WeeMingQing) [reply frontend/src/pages/Login.jsx]: Have shifted it down and removed unused packages.

**Comment by jonasgwt** [comment]
@WeeMingQing Added configuration to force dev servers to use https instead, so that access tokens and refresh tokens can be stored in a https only cookie

**Review by jonasgwt — APPROVED** [review]

**Pull request merged**